pyresample/geometry.py

```diff
@@ -269,9 +269,16 @@ def get_boundary_lonlats(self):
         return (SimpleBoundary(s1_lon.squeeze(), s2_lon.squeeze(), s3_lon.squeeze(), s4_lon.squeeze()),
                 SimpleBoundary(s1_lat.squeeze(), s2_lat.squeeze(), s3_lat.squeeze(), s4_lat.squeeze()))
 
-    def get_bbox_lonlats(self) -> tuple:
+    def get_bbox_lonlats(self, force_clockwise: bool = True) -> tuple:
         """Return the bounding box lons and lats.
 
+        Args:
+            force_clockwise:
+                Perform minimal checks and reordering of coordinates to ensure
+                that the returned coordinates follow a clockwise direction.
+                This is important for compatibility with
+                :class:`pyresample.spherical.AreaBoundary`. Default is True.
```

Comment:
I think we need more explanation on why this is important, eg `A clockwise orientation of the boundaries ensures that a the spherical bounding box covers less than half of the globe, hence that a regular swath bounding box is oriented correctly with regards to defining its inside and outside.`

Reply:
Is this the reason why? I wouldn't know as the spherical geometry stuff isn't documented so I just assumed it has to be done "because of the math". If the coordinates were counter-clockwise but the math supported/expected that then it should be fine, right? The main "issue" is that the spherical geometry expects clockwise so it must be clockwise.

Reply:
it has to go clockwise around the inside, because that's how we define inside and outside. What we want is to have the inside were the data is...

Reply:
I made an attempt. Let me know what you think. I forgot that you made your initial suggestion for what it could say, but I still stand by the point that it doesn't matter "covers less than half the globe" as this is all specific to how the spherical geometry stuff is defined.